src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
```

Comment:
The task specifies to add a data-qa attribute to the 4th link for testing, but there is no CSS selector for this attribute. Make sure to add this in the HTML and use it in CSS if styling is needed.